client/data/plugins/use-update-schedules-query.ts

```diff
@@ -42,7 +48,8 @@ export const useUpdateScheduleQuery = (
 			),
 		enabled: !! siteSlug && isEligibleForFeature,
 		retry: false,
-		refetchOnWindowFocus: false,
+		refetchOnWindowFocus: true,
+		refetchInterval: 10 * 1000,
```

Comment:
What do you think about adding this configuration from the function optional param?

It can be done in future PRs. I'm just wondering if we want to use the hook elsewhere in the code without refetching interval.